src/lib/components/csvImportBox.svelte

```diff
@@ -274,4 +323,13 @@
             background-color: var(--bgcolor-error);
         }
     }
+
+    .link-button {
+        background: none;
+        border: none;
+        padding: 0;
+        color: var(--fgcolor-error);
+        text-decoration: underline;
+        cursor: pointer;
```

Comment:
assuming these are copied from figma, we might not need all of them except color and decor maybe 🤔